.github/workflows/publish-commit-bottles.yml

```diff
@@ -46,13 +45,13 @@ jobs:
       # Workaround until the `cache` action uses the changes from
       # https://github.com/actions/toolkit/pull/580.
       - name: Unlink workspace
-        if: github.event.inputs.self_hosted != 'true'
+        if: github.event.inputs.large_runner != 'true'
```

Comment:
I mean these can probably go entirely - they exist due to non-ephemeral nature of self-hosted which will no longer be the case.

```suggestion
```

Reply:
Right, of course